src/views/assessment-plans/AssessmentPlanListView.vue

```diff
@@ -33,16 +33,14 @@
 
             <td class="py-2 px-2 text-right whitespace-nowrap">
               <div class="flex gap-2">
-                <RouterLink
-                  class="bg-white hover:bg-zinc-100 border border-ccf-300 px-4 py-1 rounded-md dark:bg-slate-800 dark:hover:bg-slate-700 dark:border-slate-700"
+                <TertiaryButton
```

Comment:
This will not work. It needs to be a router link as it links to somewhere else. (This is a note to self to fix up.)